pages/querying/schema.mdx

```diff
@@ -7,11 +7,171 @@ import { Callout } from 'nextra/components'
 
-## Schema-related queries
+## Run-time schema tracking
 
-To use schema-associated queries, start Memgraph with the `--storage-enable-schema-metadata` [configuration flag](/configuration/configuration-settings#storage) set to `True`. The flag facilitates the utilization of a specialized cache designed to store specific metadata related to the database.   
+To use run-time schema tracking, start Memgraph with the `--schema-info-enabled` [configuration flag](/configuration/configuration-settings#storage) set to `True`.Once enabled, every change to the data layout will be tracked, and a derived schema will be automatically updated. Note that enabling this feature may result in a slight performance drop; for more details, refer to [Performance penalty](#performance-penalty).
+
+### What Schema Information is Tracked?
+
+The runtime schema tracks all uniquely defined nodes and edges:
+
+* __Nodes__ are defined by their labels.
+* __Edges__ are defined by their type, and the labels of their source and destination nodes.
+
+For each unique object (node or edge), the schema tracks its count and categorizes stored properties by key and type.
+
+### Querying the Schema
+
+You can query the current schema by running  `SHOW SCHEMA INFO`. This returns a JSON representation of the schema, divided into six sections:
+
+* `nodes` - contains a set of uniquely identified nodes
+* `edges` - contains a set of uniquely identified edges
+* `node_constraints` - list of all currently defined node constraints
+* `node_indexes` - list of all currently defined node indexes
+* `edge_indexes` - list of all currently defined edge indexes
+* `enums` - list of all currently defined enums and their values
+
+#### JSON definition
+
+The schema JSON output is structured as follows:
+```
+{
+  "nodes": [{
+    "labels": [""], "count": 1,
+    "properties": [{
+      "key": "property_key_0", "count": 1, "filling_factor": 100.00,
+      "types": [{ "type": "", "count": 1}]
+    }]
+  }],
+  "edges": [{
+    "type": "", "start_node_labels": [""], "end_node_labels": [""], "count": 1,
+    "properties": [...]
+  }],
+  "node_indexes": [
+      { "labels": [""], "properties": [""], "count": 0,
+        "type": "label|label+property|label+property_point|label_text|label+property_text" }
+  ],
+  "edge_indexes": [
+      { "edge_type": "", "properties": [""],  "count": 0,
+        "type": "edge_type|edge_type+property|edge_type+property_point|label_text|label+property_text" }
+  ],
+  "node_constraints": [
+      { "type": "unique|existence|data_type", "labels": [""], "properties": [""], "data_type": "" }
+  ],
+  "enums": [
+      { "name": "", "values": [""] }
+  ]
+}
+```
+
+*Supported property types:*
+* String
+* Boolean
+* Integer
+* Float
+* List
+* Map
+* Duration
+* Date
+* LocalTime
+* LocalDateTime
+* ZonedDateTime
+* Enum::Name
+* Point2D
+* Point3D
+
+
+#### Node JSON example
+
+Consider the following queries:
+```
+CREATE (:A:B);
```

Comment:
Please include an example of a simple node with zero labels, and another simple node with a single label. After those two, then you can go into the more complex examples.